docs/changelog.md

```diff
@@ -15,7 +15,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Include `scripts/*.py` in the generated source tarballs (#1430).
 * Ensure lines after heading in loose list are properly detabbed (#1443).
 * Give smarty tree processor higher priority than toc (#1440).
-* Explicitly omit carrot (`^`) and backslash (`\`) from abbreviations (#1444).
+* Permit carrots (`^`) and square brackets (`]`) but explicitly exclude
```

Comment:
This should be "caret", not "carrot" 🤔

Reply:
🥕

Reply:
@waylan I think you missed this comment before merge ⬆️ .

Reply:
Yeah, I didn't see it until after the merge. We'll fix it next time we edit the changelog.